driver/desktop/mouse.go

```diff
@@ -11,6 +11,9 @@ const (
 
 	// RightMouseButton is the secondary button on most mouse input devices.
 	RightMouseButton
+
+	// MiddleMouseButton is the middle button on the mouse, assuming it has one.
+	MiddleMouseButton
```

Comment:
Please add Since: 2.0.0

Reply:
Sorry I should have been more specific.
Like the Deprecated: lines specified by Go, this needs to be on a newline